test/coverlet.core.tests/Coverage/CoverageTests.cs

```diff
@@ -111,7 +111,7 @@ public void SelectionStatements_If()
                 CoverageResult result = TestInstrumentationHelper.GetCoverageResult(path);
 
                 // Generate html report to check
-                // TestInstrumentationHelper.GenerateHtmlReport(result);
+                TestInstrumentationHelper.GenerateHtmlReport(result);
 
```

Comment:
Can you revert changes to this file?I'll merge package upgrade, it's ok commented, it's so as "documentation" for debugging! Thanks a lot for the fix the integration of your tool is invaluable help for me during coverage bug debugging!